### PR TITLE
Support ignoring some fields in Phabricator

### PR DESCRIPTION
**Pull request by djmitche**
In my case, I only want to see Revisions where I'm a reviewer, so this lets me disable everything else.

### CODE REVIEW

**Comment by djmitche** [comment]
It turns out Phabricator / Conduit does not support any way to get the status of a review.  So there's no way to say "Revisions that need my review".  The best option at the moment would be to screen-scrape the UI query interface.  Phabricator is weird!

File: bugwarrior/docs/services/phabricator.rst [hunk 1]
```diff
@@ -71,6 +71,22 @@ it is undefined which one will be used. To make it explicit, you can use::
...
+...................
+
+By default, any Task or Revision relating to any of the given users or projects
+will be included.  This is likely more than you want!  You can ignore some user
+relationships with the following configuration::
+
+    phabricator.ignore_cc = True       # ignore CC field
+    phabricator.ignore_author = True   # ignore Author field
+    phabricator.ignore_owner = True    # ignore Owner field (Tasks only)
+    phabricator.ignore_reviewrs = True # ignore Reviewers field (Revisions only)
```
Comment (ryneeverett) [inline bugwarrior/docs/services/phabricator.rst]: typo

**Comment by ralphbean** [comment]
Rebased and merged as ec1128c.

**Comment by djmitche** [comment]
/me waves thanks from room 9 ;)

**Comment by ralphbean** [comment]
<3

**Pull request closed without merging**